x-pack/plugins/ml/public/application/jobs/jobs_list/components/job_details/job_messages_pane.tsx

```diff
@@ -32,9 +30,18 @@ export const JobMessagesPane: FC<JobMessagesPaneProps> = ({ jobId }) => {
     }
   };
 
+  const refreshMessage = useCallback(() => fetchMessages(), [jobId]);
```

Comment:
Same here - is there a reason `fetchMessages` needs to be wrapped in another function? Since that function is only invoking `fetchMessages` it should be able to be passed directly `useCallback(fetchMessages, [jobId]);`

Again, not sure if I'm missing some context here.

Reply:
Updated in https://github.com/elastic/kibana/pull/67750/commits/9381a0d81beefa02ec6307a6fa22a797bed58197